tools/sot_siamrpn_param_search.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
```

Comment:
Please add tutorials in [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md) to show how to use this script